api/routes/units.js

```diff
@@ -21,47 +21,56 @@ router.post('/', (req, res, next) => {
 //  [X] [GET /api/v1/units]
 //  [X] [GET /api/v1/units?kind=[kind]]
 //  [X] [GET /api/v1/units?floor=[integer]]
-//  [ ] [GET /api/v1/units?occupied=[true/false]]
-
-// router.get('/', async (req, res, next) => {
-//         const status = 200
-//         if (req.query) {
-//             const { kind, floor, occupied} = req.query
-//             if (kind) {
-//                 console.log(kind)
-//             } else if (floor) {
-//                 console.log(floor)
-//             } else if (occupied) {
-//                 console.log(occupied)
-//             } else {
-//                 console.log('no query')
-//             }
-
-//         } else {
-//             console.log('no query')
-//         }
-//         const response = await Unit.find(req.query).select('-created_at -updated_at -__v')//.select('_id company kind floor special_monthly_offer')
-//         res.json({ status, response })
-// })
+//  [X] [GET /api/v1/units?occupied=[true/false]]
 
 router.get('/', async (req, res, next) => {
     const status = 200
-    if (req.query) {
-        const response = await Unit.find(req.query).select('_id company kind floor special_monthly_offer')
-        res.json({
-            status,
-            response
-        })
-    } else {
-        const response = await Unit.find().select('_id company kind floor special_monthly_offer')
-        res.json({
-            status,
-            response
-        })
+    try {
+        if (req.query) {
```

Comment:
This will always return `true`, as `req.query` is an object and therefore truthy (even if it's empty).